sys.version_info(major=3, minor=6, micro=0, releaselevel='final', serial=0)
----- PARAMETERS -----
DETERMINISTIC: 1
DemandIntercept: 300
EnableProfiling: False
FixedCostOfImitation: 50
FixedCostOfInnovation: 100
FixedProductionCost: 200
InitialAttractionForImitation: 10
InitialAttractionForInnovation: 10
InitialAttractionForNoResearch: 10
InitialAttractionForResearch: 10
LogLevel: {'Console': ['INFO'], 'File': ['INFO']}
MaxMagnituteOfChangeInTechEnv: 16
MeanMarketSize: 4
MinMarketSize: 0.1
NumberOfParallelProcesses: 72
NumberOfPotentialEntrants: 40
NumberOfSimulations: 1
NumberOfTasks: 96
PeriodOfHalfTurn: 500
PeriodStartOfTechChange: 0
PeriodsOfConstantDemand: 2000
RateOfChangeInTechEnv: 0.2
RateOfMeanMarketSizeGrowth: 0
RateOfPersistenceInDemand: 0.97
STOCHASTIC: 0
StartupWealth: 0
ThresholdNetWealthForSurvival: 0
TimeHorizon: 5000
TypeOfCycle: None
WaveAmplitude: 2

Executing 1 simulations

[SIM 1] STARTING SIMULATION
[SIM 1][PERIOD 1] --------------------- PROCESSING PERIOD ---------------------
Error
Traceback (most recent call last):
  File "/Users/jdimas/GitHub/ACEModel/Simulation.py", line 22, in runSimulation
    industry.processPeriod()
  File "/Users/jdimas/GitHub/ACEModel/model/industry.py", line 36, in processPeriod
    self.processShutdownDecisions()
  File "/Users/jdimas/GitHub/ACEModel/model/industry.py", line 176, in processShutdownDecisions
    if leastEfficient.decideIfDeactivates():
  File "/Users/jdimas/GitHub/ACEModel/model/firm.py", line 85, in decideIfDeactivates
    self.updateOutput()
  File "/Users/jdimas/GitHub/ACEModel/model/firm.py", line 44, in updateOutput
    assert self.output == self.industry.demand.marketSize * (1 / (len(self.industry.incumbentFirms) + 1) * (Parameters.DemandIntercept + self.industry.currentActiveSumOfMC) - self.MC)
AssertionError
Error
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/Users/jdimas/GitHub/ACEModel/Simulation.py", line 34, in runSimulation
    raise e
  File "/Users/jdimas/GitHub/ACEModel/Simulation.py", line 22, in runSimulation
    industry.processPeriod()
  File "/Users/jdimas/GitHub/ACEModel/model/industry.py", line 36, in processPeriod
    self.processShutdownDecisions()
  File "/Users/jdimas/GitHub/ACEModel/model/industry.py", line 176, in processShutdownDecisions
    if leastEfficient.decideIfDeactivates():
  File "/Users/jdimas/GitHub/ACEModel/model/firm.py", line 85, in decideIfDeactivates
    self.updateOutput()
  File "/Users/jdimas/GitHub/ACEModel/model/firm.py", line 44, in updateOutput
    assert self.output == self.industry.demand.marketSize * (1 / (len(self.industry.incumbentFirms) + 1) * (Parameters.DemandIntercept + self.industry.currentActiveSumOfMC) - self.MC)
AssertionError
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/jdimas/GitHub/ACEModel/Simulation.py", line 62, in <module>
    multiAggregateData.addListOfResults(listOfResults) # Add result to the aggregate list (this will be used to calculate the means)
  File "/Users/jdimas/GitHub/ACEModel/model/aggregate_data.py", line 147, in addListOfResults
    for result in listOfResults:
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/multiprocessing/pool.py", line 699, in next
    raise value
AssertionError
